src/web/layouts/ShowcaseLayout.tsx

```diff
@@ -209,6 +209,24 @@ const PositionHeadline = ({
     }
 };
 
+// The advert is stuck to the top of the container as we scroll
+// until we hit the bottom of the wrapper that contains
+// the top banner and the header/navigation
+// We apply sticky positioning and z-indexes, the stickAdWrapper and headerWrapper
+// classes are tightly coupled.
+
+const stickyAdWrapper = css`
+    background-color: white;
+    border-bottom: 0.0625rem solid ${palette.neutral[86]};
+    position: sticky;
+    top: 0;
+    z-index: 2;
+`;
+
+const headerWrapper = css`
+    position: relative;
+    z-index: 1;
+`;
```

Comment:
I'll be adding an additional layout file soon so at that point we can look at moving this up but it makes sense to keep it local for now